From: Aurelien Jarno <aurel32@debian.org>
Date: Fri, 29 May 2009 03:28:45 -0600
Subject: remaining kFreeBSD hackery for building.

Addresses-Debian-Bug: #527384
Signed-off-by: LaMont Jones <lamont@debian.org>

Conflicts:
	login-utils/Makefile.am
	login-utils/agetty.c
	sys-utils/ipcs.c
---
 sys-utils/ipcs.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/sys-utils/ipcs.c b/sys-utils/ipcs.c
index 8ed5698..9ce5366 100644
--- a/sys-utils/ipcs.c
+++ b/sys-utils/ipcs.c
@@ -461,8 +461,10 @@ static void do_msg (char format, int unit)
 			return;
 		}
 		printf (_("------ Messages Status --------\n"));
+#ifndef __FreeBSD_kernel__
 		printf (_("allocated queues = %d\n"), msginfo.msgpool);
 		printf (_("used headers = %d\n"), msginfo.msgmap);
+#endif
 		ipc_print_size(unit, _("used space"), msginfo.msgtql,
 			       unit == IPC_UNIT_DEFAULT ? _(" bytes\n") : "\n", 0);
 		return;
